submit/controllers/ui/new/authorship.py

```diff
@@ -60,29 +59,21 @@ def authorship(method: str, params: MultiDict, session: Session,
         'client': client,
     }
 
-    if method == 'POST':
-        if not form.validate():
-            raise BadRequest(response_data)
+    if method == 'POST' and form.validate():
         value = (form.authorship.data == form.YES)
-
         # No need to do this more than once.
         if submission.submitter_is_author != value:
             command = ConfirmAuthorship(creator=submitter, client=client,
                                         submitter_is_author=value)
-
-            if not validate_command(form, command, submission, 'authorship'):
-                raise BadRequest(response_data)
-
-            try:
-                # Create ConfirmAuthorship event
```

Comment:
It would be helpful to retain comments like these (if they are still accurate).

Reply:
The ConfirmAuthorship is created a couple lines above. I fee the comment is redundant.